main/src/com/google/refine/model/Recon.java

```diff
@@ -69,7 +69,8 @@ static public enum Judgment {
         @JsonProperty("none")
         None, @JsonProperty("matched")
         Matched, @JsonProperty("new")
-        New
+        New, @JsonProperty("errors")
```

Comment:
I'd adjust the JSON serialization of this field to be just `error`, not `errors`
```suggestion
        New, @JsonProperty("error")
```